.github/workflows/prerelease.yml

```diff
@@ -0,0 +1,62 @@
+name: Prerelease 🚀
+
+on:
+  push:
+    branches:
+      # Target main branch
+      - "main"
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  prerelease:
+    name: Changests Prerelease
+    if: github.repository == 'latticexyz/mud'
+    runs-on: ubuntu-latest
+    # Permissions necessary for Changesets to push a new branch and open PRs
+    # (for automated Version Packages PRs), and request the JWT for provenance.
+    # More info: https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect#adding-permissions-settings
+    permissions:
+      contents: write
+      pull-requests: write
+      id-token: write
+    steps:
+      - uses: actions/checkout@v3
+        name: Checkout Repo
+        with:
+          submodules: recursive
+
+      - name: "Setup"
+        uses: ./.github/actions/setup
+        
+      - name: Generate gas reports
+        run: pnpm gas-report
+
+      - name: Outdated files, run `pnpm gas-report` and commit them
+        uses: ./.github/actions/require-empty-diff
```

Comment:
i think we can skip this for the release action (since this only runs on main, where checks should already have passed in the PRs that got merged into main, and even if the gas reports are out of date it's not blocking for the release)